### PR TITLE
(BOLT-1345) Add sidebar to website

### PR DESCRIPTION
**Pull request by nicklewis**
This creates a new navigation sidebar on the website that includes links
to each of the lab exercises. To facilitate this, the lab exercises have
been moved into a new jekyll "collection" called labs and are
dynamically included in the list.

### CODE REVIEW

**Comment by nicklewis** [comment]
This should be live for previewing on https://nicklewis.github.io/bolt/ soon

**Comment by nicklewis** [comment]
Perhaps *even more importantly*, I fixed the favicon!

File: docs/_layouts/default.html [hunk 1]
```diff
@@ -18,21 +18,39 @@
...
     <link rel="stylesheet" href="{{ '/assets/css/style.css?v=' | append: site.github.build_revision | relative_url }}">
-    <link rel="icon" href="/favicon.ico" type="image/x-icon" />
+    <link rel="icon" href="{{ '/favicon.ico' | relative_url }}" type="image/x-icon" />
   </head>
-  <body>
+  <body class="container">
+    <nav class="sidebar">
+      <ul class="nav-list">
+        <li>
+          <span class="nav-header">Bolt Hands-on Lab</span>
```
Comment (lucywyman) [inline docs/_layouts/default.html]: It'd be awesome if this included the logo somehow! Or maybe just add the logo above or below it?

**Comment by nicklewis** [comment]
Updated with an attempt at including the logo

**Comment by lucywyman** [comment]
Assuming you saw https://github.com/nicklewis/bolt/pull/1 and http://lucywyman.me/bolt/ - I'm good with this if we don't want to do all the fancy CSS!

**Pull request merged**